libraries/RC_Channel/RC_Channel.h

```diff
@@ -601,7 +601,10 @@ class RC_Channels {
     // get last aux cached value for scripting. Returns false if never set, otherwise 0,1,2
     bool get_aux_cached(RC_Channel::aux_func_t aux_fn, uint8_t &pos);
 #endif
-    
+
+    // get failsafe timeout in seconds
+    float get_fs_timeout() const { return MAX(_fs_timeout, 0.1); }
```

Comment:
peter wanted get_fs_timeout_ms() as uint32_t

Reply:
changed to _ms() and returns uint32_t.